demosplan/DemosPlanCoreBundle/ResourceTypes/StatementFragmentResourceType.php

```diff
@@ -159,18 +159,18 @@ public function isReferencable(): bool
         return false;
     }
 
-    public function getAccessCondition(): PathsBasedInterface
+    protected function getAccessConditions(): array
     {
         $procedure = $this->currentProcedureService->getProcedure();
         if (null === $procedure) {
-            return $this->conditionFactory->false();
+            return [$this->conditionFactory->false()];
         }
 
-        return $this->conditionFactory->allConditionsApply(
+        return [
             $this->conditionFactory->propertyHasValue($procedure->getId(), $this->procedure->id),
             $this->conditionFactory->propertyHasValue($procedure->getId(), $this->statement->procedure->id),
-            $this->conditionFactory->propertyHasValue(false, $this->deleted),
+            $this->conditionFactory->propertyHasValue(false, $this->deleted), // FIXME: cdr: this should not work, there is no `deleted` property in `StatementFragment` nor is `deleted` set up as alias in this resource type
```

Comment:
Please listen to the fix me and fix it :)

Reply:
done in a078b494f949431f495041dbc9ef6c9c8f667c6e